packages/experimental/Checkbox/src/Checkbox.styling.ts

```diff
@@ -26,21 +29,15 @@ export const stylingSettings: UseStylingOptions<CheckboxProps, CheckboxSlotProps
           alignSelf: 'flex-start',
           backgroundColor: tokens.backgroundColor,
           ...borderStyles.from(tokens, theme),
-          padding: 4,
+          padding: tokens.padding,
```

Comment:
What happens if the user doesn't pass in padding? We go from defaulting the padding from 4 to whatever the user passes in. Will this have any negative effect on the base Checkbox styling?